superset/config.py

```diff
@@ -1085,10 +1085,10 @@ def CSV_TO_HIVE_UPLOAD_DIRECTORY_FUNC(
 # use the "engine_name" attribute of the corresponding DB engine spec
 # in `superset/db_engine_specs/`.
 PREFERRED_DATABASES: List[str] = [
-    # "PostgreSQL",
-    # "Presto",
-    # "MySQL",
-    # "SQLite",
+    "PostgreSQL",
+    "Presto",
+    "MySQL",
+    "SQLite",
     # etc.
```

Comment:
Do we wanna curate this list better? I just wrote these of the top of my head.

Reply:
I think these are fine for now

@yousoph what do you think?